### PR TITLE
fix: add labels to examples

### PR DESCRIPTION
**Pull request by Andras-Csanyi**
## PR summary
This PR adds labels to example output, so that it is easier to identify which section of the output belongs to which method.

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

File: casemanagementv1/case_management_v1_examples_test.go [hunk 1]
```diff
@@ -28,8 +28,6 @@ import (
 
 	"github.com/IBM/go-sdk-core/v5/core"
 	"github.com/IBM/platform-services-go-sdk/casemanagementv1"
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
```
Comment (pyrooka) [inline casemanagementv1/case_management_v1_examples_test.go:32]: These imports are necessary. Without them cannot run the tests (Describe, Expect, It, etc...)

File: casemanagementv1/case_management_v1_examples_test.go [hunk 2]
```diff
@@ -271,7 +269,7 @@ var _ = Describe(`CaseManagementV1 Examples Tests`, func() {
 				panic(err)
 			}
 			b, _ := json.MarshalIndent(watchlist, "", "  ")
-			fmt.Println(string(b))
+			fmt.Printf("\nRemoveWatchlist() result:\n &s \n", string(b))
```
Comment (pyrooka) [inline casemanagementv1/case_management_v1_examples_test.go:272]: Typo.
```suggestion
			fmt.Printf("\nRemoveWatchlist() result:\n %s \n", string(b))
```

File: atrackerv1/atracker_v1_examples_test.go [hunk 3]
```diff
@@ -115,7 +113,7 @@ var _ = Describe(`AtrackerV1 Examples Tests`, func() {
 				panic(err)
 			}
 			b, _ := json.MarshalIndent(target, "", "  ")
-			fmt.Println(string(b))
+			fmt.Printf("\nCreateTarget() result:\n %s \n", string(b))
```
Comment (pyrooka) [inline atrackerv1/atracker_v1_examples_test.go:116]: Using space around the `%s` causes "not that pretty" indentation:
```
UploadFile() result:
 {
  "id": "xxx",
  "filename": "example.log",
  "size_in_bytes": 42,
  "created_at": "2021-03-23 16:41:50",
  "url": "xxx"
}
```
```suggestion
			fmt.Printf("\nCreateTarget() result:\n%s\n", string(b))
```

**Review by pyrooka — CHANGES_REQUESTED** [review]
Apart from the requested changes looks good. What do you think about the things I've mentioned?

**Review by padamstx — APPROVED** [review]
LGTM

**Review by pyrooka — APPROVED** [review]
👍

**Pull request merged**